FWCore/MessageService/plugins/MessageLogger.h

```diff
@@ -193,4 +190,4 @@ namespace edm {
 
 }  // namespace edm
 
-#endif  // FWCore_MessageService_MessageLogger_h
+#endif  // FWCore_MessageService_service_MessageLogger_h
```

Comment:
```suggestion
#endif  // FWCore_MessageService_plugins_MessageLogger_h
```
to be consistent with the guards